### PR TITLE
Updates foxes so they aren't strange when they do their simple-mob speech emotes

### PR DESCRIPTION
**Pull request by VaelophisNyx**
# Document the changes in your pull request

What does the fox say? Well for one not...."geckers"??
observe: https://youtu.be/C82XoWjSqLk
see also: https://youtu.be/QamQFnZjxeA
and: https://youtu.be/N3xGCRsEEiY

also makes fennecs move more often. This is realistic as they're tiny balls of energy and screams IRL

# Changelog

:cl:  
tweak: What does the fox say? it says AAAAAAAAAAAAAA
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/friendly/fox.dm [hunk 1]
```diff
@@ -6,23 +6,23 @@
 	icon_state = "fox"
 	icon_living = "fox"
 	icon_dead = "fox_dead"
-	speak = list("Ack-Ack","Ack-Ack-Ack-Ackawoooo","Geckers","Awoo","Tchoff")
-	speak_emote = list("geckers", "barks")
-	emote_hear = list("howls.","barks.")
+	speak = list("AAAAAAAAAAAAAAAAAAAA","Yip","Hehehehehe")
+	speak_emote = list("screamss")
```
Comment (Sniblet) [inline code/modules/mob/living/simple_animal/friendly/fox.dm]: screams. also "Autumn screams, 'Yip'" is kind of hard to picture - don't foxes have other tones of voice?

File: code/modules/mob/living/simple_animal/friendly/fox.dm [hunk 2]
```diff
@@ -6,23 +6,23 @@
...
 	response_disarm = "gently pushes aside"
 	response_harm = "kicks"
 	attack_vis_effect = ATTACK_EFFECT_BITE
 	gold_core_spawnable = FRIENDLY_SPAWN
 	can_be_held = TRUE
 	do_footstep = TRUE
-	wuv_happy = "yaps happily!"
-	wuv_angy = "growls!"
+	wuv_happy = "screams happily!"
+	wuv_angy = "screams angerly!"
```
Comment (Sniblet) [inline code/modules/mob/living/simple_animal/friendly/fox.dm]: angrily? unless this is deliberate cute misspelling?

File: code/modules/mob/living/simple_animal/friendly/fox.dm (hunk 1 above)
Comment (VaelophisNyx) [reply code/modules/mob/living/simple_animal/friendly/fox.dm]: not exactly, at least not ones that like...make sense in a sentence

File: code/modules/mob/living/simple_animal/friendly/fox.dm (hunk 2 above)
Comment (VaelophisNyx) [reply code/modules/mob/living/simple_animal/friendly/fox.dm]: it was a typo ye, not intentional, will fix soon

**Review by Sniblet — APPROVED** [review]
I don't know what reviewing means but uhh yeah thumbs up cool work bro!! The grammar's clean now!! Woo!!

**Review by Bibby0110 — APPROVED** [review]

**Pull request merged**